apps/l3dtool/l3dtool.cpp

```diff
@@ -781,6 +814,16 @@ int ExtractFile(const Arguments::Extract& args)
 	memcpy(indexBuffer.data.data(), l3d.GetIndices().data(), sizeOfIndices);
 	gltf.buffers.push_back(indexBuffer);
 
+	// TODO(raffclar): tiny_gltf has not yet added support for serialising binary data. Using strings with base64
```

Comment:
For to-do in 3rd parties, we should open and issue in their repo and one in ours linking theirs.
Then this comment should be updated with a link to the tiny_gltf issue.

Reply:
Agreed. I'll get on it

Reply:
https://github.com/syoyo/tinygltf/issues/360